pkg/apis/operator/v1alpha1/tektonhub_types.go

```diff
@@ -120,6 +125,10 @@ type TektonHubStatus struct {
 	// The current installer set name
 	// +optional
 	HubInstallerSet map[string]string `json:"hubInstallerSets,omitempty"`
+
+	// The Base64 Encode data info of the Custom Logo
```

Comment:
I believe you are getting input from user via CR.
I just noticed that, you have added `CustomLogo` under `TektonHubStatus`
Status is generated by controller.
`CustomLogo` should go under `TektonHubSpec` 

@vdemeester please confirm